Bender.yml

```diff
@@ -44,7 +52,13 @@ sources:
   - rtl/cluster_clock_gate.sv
   - rtl/cluster_event_map.sv
   - rtl/cluster_timer_wrap.sv
-  - rtl/dmac_wrap.sv
+  - rtl/obi_pulp_adapter.sv
+  - target: mchan
```

Comment:
The proliferation of targets with potentially misleading names is a bit dangerous. I think this is safe, but maybe `pulp_cluster_mchan` would be a better name.